.circleci/config.yml

```diff
@@ -312,6 +313,75 @@ jobs:
     <<: *test-defaults
     environment:
       - TEST_TARGET=sanity
+  build-waterfall:
+    <<: *defaults
+    working_directory: ~/
+    steps:
+      - checkout:
+          path: emscripten/
+      - run:
+          name: install package dependencies
+          command: |
+            apt-get update -q
+            apt-get install -q -y python3 cmake
+      - run:
+          name: install emsdk
+          command: |
+            wget https://github.com/juj/emsdk/archive/master.tar.gz
+            tar -xf master.tar.gz
+            cd emsdk-master
+            ./emsdk --notty update-tags
+            export WATERFALL_LKGR=`python -c "print [x.split('\"')[3] for x in open('upstream/lkgr.json').readlines() if '\"build\"' in x][0]"`
```

Comment:
That's pretty un-ergonomic yeah

Reply:
I couldn't find a better way :) Is there?

Reply:
Not that I'm aware of! Moreso that's a comment on how clunky it is to get upstream-llvm via emsdk, and how automation-unfriendly it is.
In principle this should be handled in emsdk itself, say `./emsdk install upstream-lkgr-64bit`, but I couldn't figure out how to get it to do that easily.

Reply:
Hmm yeah, the emsdk should really support this. We do have `emsdk install latest`, so we should add something like `latest-upstream` perhaps.